src/browser/context-menu-template.js

```diff
@@ -0,0 +1,99 @@
+var reload = function(item, focusedWindow) {
+  if (focusedWindow) {
+    focusedWindow.webContents.reload();
+  }
+};
+
+var windows_linux_template = [
+  {
+    label: 'Undo',
+    accelerator: 'CmdOrCtrl+Z',
+    role: 'undo:'
+  },
+  {
+    label: 'Redo',
+    accelerator: 'Shift+CmdOrCtrl+Z',
+    role: 'redo:'
+  },
+  {
+    type: 'separator'
+  },
+  {
+    label: 'Cut',
+    accelerator: 'CmdOrCtrl+X',
+    role: 'cut:'
+  },
+  {
+    label: 'Copy',
+    accelerator: 'CmdOrCtrl+C',
+    role: 'copy:'
+  },
+  {
+    label: 'Paste',
+    accelerator: 'CmdOrCtrl+V',
+    role: 'paste:'
+  },
+  {
+    label: 'Select All',
+    accelerator: 'CmdOrCtrl+A',
+    role: 'selectAll:'
+  },
+  {
+    type: 'separator'
+  },
+  {
+    label: 'Reload',
+    accelerator: 'CmdOrCtrl+R',
+    click: reload
+  }
+];
+
+var osx_template = [
+  {
+    label: 'Undo',
+    accelerator: 'CmdOrCtrl+Z',
+    selector: 'undo:'
+  },
+  {
+    label: 'Redo',
+    accelerator: 'Shift+CmdOrCtrl+Z',
+    selector: 'redo:'
+  },
+  {
+    type: 'separator'
+  },
+  {
+    label: 'Cut',
+    accelerator: 'CmdOrCtrl+X',
+    selector: 'cut:'
+  },
+  {
+    label: 'Copy',
+    accelerator: 'CmdOrCtrl+C',
+    selector: 'copy:'
+  },
+  {
+    label: 'Paste',
+    accelerator: 'CmdOrCtrl+V',
+    selector: 'paste:'
+  },
+  {
+    label: 'Select All',
+    accelerator: 'CmdOrCtrl+A',
+    selector: 'selectAll:'
+  },
+  {
+    type: 'separator'
+  },
+  {
+    label: 'Reload',
+    accelerator: 'CmdOrCtrl+R',
+    click: reload
+  }
+];
+
+if (process.platform === 'darwin') {
```

Comment:
If you change this condition so it is false you'll see that the use of `role` instead of `selector` in the menu items does not work on OSx. At least that's what I saw. I was not able to test for Windows.